--------------------Moonraker Log Start--------------------
Git Version: v0.6.0-176-g5c05afa

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
enable_debug_logging = False
config_path = /home/klippy/.config

[authorization]
enabled = True
api_key_file = /home/klippy/.moonraker_api_key
cors_domains = 
	http://*
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	192.168.100.0/24
	FE80::/10
	::1/128

[octoprint_compat]

[history]

[power tasmota]
type = tasmota
address = 192.168.0.13

[update_manager]
enable_auto_refresh = True

[update_manager client ratrig-theme]
type = git_repo
path = /home/klippy/.config/.theme
origin = https://github.com/Rat-Rig/mainsail-theme.git

[update_manager client vcore3]
type = git_repo
path = /home/klippy/.config/v-core-3
origin = https://github.com/Rat-Rig/v-core-3-klipper-config.git

[update_manager client klipperscreen]
type = git_repo
path = /home/klippy/klipperscreen
origin = https://github.com/jordanruthe/KlipperScreen.git

[update_manager client klipper]
type = git_repo
path = /home/klippy/klipper
origin = https://github.com/KevinOConnor/klipper.git

[system_args]
configfile = /home/klippy/.config/moonraker.conf
logfile = /logs/moonraker.log
software_version = v0.6.0-176-g5c05afa

#################################################################

System Info:

***cpu_info***
  cpu_count: 4
  bits: 64bit
  processor: aarch64
  cpu_desc: 
  hardware_desc: ODROID-C2
  model: 
  total_memory: 1767296
  memory_units: kB

***sd_info***
  manufacturer_id: 15
  manufacturer: Unknown
  oem_id: 0100
  product_name: BGND3
  product_revision: 5.2
  serial_number: 017a50b4
  manufacturer_date: 3/2147
  capacity: Unknown
  total_bytes: 0

***distribution***
  name: Debian GNU/Linux 10 (buster)
  id: debian
  version: 10
  version_parts: {'major': '10', 'minor': '', 'build_number': ''}
  like: 
  codename: buster
2021-07-02 00:13:06,424 [websockets.py:remove_websocket()] - Websocket Removed: 547568683808
2021-07-02 00:13:06,426 [websockets.py:_process_messages()] - Websocket closed while writing: 547568683808
2021-07-02 00:21:36,630 [shell_command.py:_create_subprocess()] - shell_command: Command (sudo DEBIAN_FRONTEND=noninteractive apt-get update) failed
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/components/shell_command.py", line 253, in _create_subprocess
    transport, protocol = await loop.subprocess_exec(
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1661, in subprocess_exec
    transport = await self._make_subprocess_transport(
  File "/usr/local/lib/python3.9/asyncio/unix_events.py", line 197, in _make_subprocess_transport
    transp = _UnixSubprocessTransport(self, protocol, args, shell,
  File "/usr/local/lib/python3.9/asyncio/base_subprocess.py", line 36, in __init__
    self._start(args=args, shell=shell, stdin=stdin, stdout=stdout,
  File "/usr/local/lib/python3.9/asyncio/unix_events.py", line 789, in _start
    self._proc = subprocess.Popen(
  File "/usr/local/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'sudo'
2021-07-02 00:21:36,643 [shell_command.py:_create_subprocess()] - shell_command: Command (sudo DEBIAN_FRONTEND=noninteractive apt-get update) failed
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/components/shell_command.py", line 253, in _create_subprocess
    transport, protocol = await loop.subprocess_exec(
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1661, in subprocess_exec
    transport = await self._make_subprocess_transport(
  File "/usr/local/lib/python3.9/asyncio/unix_events.py", line 197, in _make_subprocess_transport
    transp = _UnixSubprocessTransport(self, protocol, args, shell,
  File "/usr/local/lib/python3.9/asyncio/base_subprocess.py", line 36, in __init__
    self._start(args=args, shell=shell, stdin=stdin, stdout=stdout,
  File "/usr/local/lib/python3.9/asyncio/unix_events.py", line 789, in _start
    self._proc = subprocess.Popen(
  File "/usr/local/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'sudo'
2021-07-02 00:21:36,657 [shell_command.py:_create_subprocess()] - shell_command: Command (sudo DEBIAN_FRONTEND=noninteractive apt-get update) failed
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/components/shell_command.py", line 253, in _create_subprocess
    transport, protocol = await loop.subprocess_exec(
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1661, in subprocess_exec
    transport = await self._make_subprocess_transport(
  File "/usr/local/lib/python3.9/asyncio/unix_events.py", line 197, in _make_subprocess_transport
    transp = _UnixSubprocessTransport(self, protocol, args, shell,
  File "/usr/local/lib/python3.9/asyncio/base_subprocess.py", line 36, in __init__
    self._start(args=args, shell=shell, stdin=stdin, stdout=stdout,
  File "/usr/local/lib/python3.9/asyncio/unix_events.py", line 789, in _start
    self._proc = subprocess.Popen(
  File "/usr/local/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'sudo'
2021-07-02 00:21:36,658 [update_manager.py:refresh()] - Error Refreshing System Packages
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/components/update_manager.py", line 1470, in refresh
    await self.cmd_helper.run_cmd(
  File "/home/klippy/moonraker/moonraker/components/update_manager.py", line 413, in run_cmd
    raise self.server.error("Shell Command Error")
utils.ServerError: Shell Command Error
2021-07-02 00:21:36,688 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker status -u no) successfully finished
2021-07-02 00:21:36,707 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker config --get branch.master.remote) successfully finished
2021-07-02 00:21:36,727 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker branch --list) successfully finished
2021-07-02 00:21:37,585 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker fetch origin --prune --progress) successfully finished
2021-07-02 00:21:37,605 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker remote get-url origin) successfully finished
2021-07-02 00:21:37,636 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker rev-parse HEAD) successfully finished
2021-07-02 00:21:37,657 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker rev-parse origin/master) successfully finished
2021-07-02 00:21:37,697 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker describe --always --tags --long --dirty) successfully finished
2021-07-02 00:21:37,734 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/moonraker describe origin/master --always --tags --long) successfully finished
2021-07-02 00:21:37,734 [update_manager.py:log_repo_info()] - Git Repo moonraker Detected:
Owner: Arksine
Path: /home/klippy/moonraker
Remote: origin
Branch: master
Remote URL: https://github.com/Arksine/moonraker.git
Current Commit SHA: 5c05afae42414bb591bfd580206466fffe14b2b1
Upstream Commit SHA: 5c05afae42414bb591bfd580206466fffe14b2b1
Current Version: v0.6.0-176
Upstream Version: v0.6.0-176
Is Dirty: False
Is Detached: False
Commits Behind: 0
2021-07-02 00:21:37,735 [update_manager.py:_log_info()] - Repo moonraker: Validity check for git repo passed
2021-07-02 00:21:37,761 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme status -u no) successfully finished
2021-07-02 00:21:37,795 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme config --get branch.master.remote) successfully finished
2021-07-02 00:21:37,817 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme branch --list) successfully finished
2021-07-02 00:21:38,704 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme fetch origin --prune --progress) successfully finished
2021-07-02 00:21:38,725 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme remote get-url origin) successfully finished
2021-07-02 00:21:38,745 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme rev-parse HEAD) successfully finished
2021-07-02 00:21:38,765 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme rev-parse origin/master) successfully finished
2021-07-02 00:21:38,788 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme describe --always --tags --long --dirty) successfully finished
2021-07-02 00:21:38,810 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/.theme describe origin/master --always --tags --long) successfully finished
2021-07-02 00:21:38,811 [update_manager.py:log_repo_info()] - Git Repo ratrig-theme Detected:
Owner: Rat-Rig
Path: /home/klippy/.config/.theme
Remote: origin
Branch: master
Remote URL: https://github.com/Rat-Rig/mainsail-theme
Current Commit SHA: 723064e3edee72fde1f3b0e92fff408efe6b7562
Upstream Commit SHA: 723064e3edee72fde1f3b0e92fff408efe6b7562
Current Version: v0.1.0-0
Upstream Version: v0.1.0-0
Is Dirty: False
Is Detached: False
Commits Behind: 0
2021-07-02 00:21:38,811 [update_manager.py:_log_info()] - Repo ratrig-theme: Validity check for git repo passed
2021-07-02 00:21:38,835 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 status -u no) successfully finished
2021-07-02 00:21:38,855 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 config --get branch.master.remote) successfully finished
2021-07-02 00:21:38,881 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 branch --list) successfully finished
2021-07-02 00:21:39,784 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 fetch origin --prune --progress) successfully finished
2021-07-02 00:21:39,804 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 remote get-url origin) successfully finished
2021-07-02 00:21:39,825 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 rev-parse HEAD) successfully finished
2021-07-02 00:21:39,846 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 rev-parse origin/master) successfully finished
2021-07-02 00:21:39,889 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 describe --always --tags --long --dirty) successfully finished
2021-07-02 00:21:39,913 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/.config/v-core-3 describe origin/master --always --tags --long) successfully finished
2021-07-02 00:21:39,913 [update_manager.py:log_repo_info()] - Git Repo vcore3 Detected:
Owner: Rat-Rig
Path: /home/klippy/.config/v-core-3
Remote: origin
Branch: master
Remote URL: https://github.com/Rat-Rig/v-core-3-klipper-config
Current Commit SHA: f6460ec8c71d1e378647bea01618e827d6266a09
Upstream Commit SHA: f6460ec8c71d1e378647bea01618e827d6266a09
Current Version: v0.1.2-14
Upstream Version: v0.1.2-14
Is Dirty: False
Is Detached: False
Commits Behind: 0
2021-07-02 00:21:39,914 [update_manager.py:_log_info()] - Repo vcore3: Validity check for git repo passed
2021-07-02 00:21:39,939 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen status -u no) successfully finished
2021-07-02 00:21:39,959 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen config --get branch.master.remote) successfully finished
2021-07-02 00:21:39,980 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen branch --list) successfully finished
2021-07-02 00:21:40,896 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen fetch origin --prune --progress) successfully finished
2021-07-02 00:21:40,915 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen remote get-url origin) successfully finished
2021-07-02 00:21:40,934 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen rev-parse HEAD) successfully finished
2021-07-02 00:21:40,954 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen rev-parse origin/master) successfully finished
2021-07-02 00:21:40,991 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen describe --always --tags --long --dirty) successfully finished
2021-07-02 00:21:41,023 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen describe origin/master --always --tags --long) successfully finished
2021-07-02 00:21:41,044 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen log 912ecea3e3b9bed930d9304f667ead51d06143f8..origin/master --format="sha:%H%x1Dauthor:%an%x1Ddate:%ct%x1Dsubject:%s%x1Dmessage:%b%x1E" --max-count=100) successfully finished
2021-07-02 00:21:41,064 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipperscreen show-ref --tags -d) successfully finished
2021-07-02 00:21:41,065 [update_manager.py:log_repo_info()] - Git Repo klipperscreen Detected:
Owner: jordanruthe
Path: /home/klippy/klipperscreen
Remote: origin
Branch: master
Remote URL: https://github.com/jordanruthe/KlipperScreen
Current Commit SHA: 912ecea3e3b9bed930d9304f667ead51d06143f8
Upstream Commit SHA: f68f9c642b9ecb184ed2610327203c295c1bc54b
Current Version: v0.1.6-50
Upstream Version: v0.1.6-51
Is Dirty: False
Is Detached: False
Commits Behind: 1
2021-07-02 00:21:41,065 [update_manager.py:_log_info()] - Repo klipperscreen: Validity check for git repo passed
2021-07-02 00:21:41,091 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper status -u no) successfully finished
2021-07-02 00:21:41,110 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper config --get branch.master.remote) successfully finished
2021-07-02 00:21:41,139 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper branch --list) successfully finished
2021-07-02 00:21:42,106 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper fetch origin --prune --progress) successfully finished
2021-07-02 00:21:42,126 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper remote get-url origin) successfully finished
2021-07-02 00:21:42,156 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper rev-parse HEAD) successfully finished
2021-07-02 00:21:42,175 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper rev-parse origin/master) successfully finished
2021-07-02 00:21:42,308 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper describe --always --tags --long --dirty) successfully finished
2021-07-02 00:21:42,432 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper describe origin/master --always --tags --long) successfully finished
2021-07-02 00:21:42,453 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper log 430578957f37f42815ffea900cbc3459d6fd0888..origin/master --format="sha:%H%x1Dauthor:%an%x1Ddate:%ct%x1Dsubject:%s%x1Dmessage:%b%x1E" --max-count=100) successfully finished
2021-07-02 00:21:42,474 [shell_command.py:_check_proc_success()] - Command (git -C /home/klippy/klipper show-ref --tags -d) successfully finished
2021-07-02 00:21:42,475 [update_manager.py:log_repo_info()] - Git Repo klipper Detected:
Owner: KevinOConnor
Path: /home/klippy/klipper
Remote: origin
Branch: master
Remote URL: https://github.com/KevinOConnor/klipper.git
Current Commit SHA: 430578957f37f42815ffea900cbc3459d6fd0888
Upstream Commit SHA: ab8c571070506e037e00d28a1adb9a1ad9592aed
Current Version: v0.9.1-591
Upstream Version: v0.9.1-592
Is Dirty: False
Is Detached: False
Commits Behind: 1
2021-07-02 00:21:42,475 [update_manager.py:_log_info()] - Repo klipper: Validity check for git repo passed
2021-07-02 00:46:36,486 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.41
2021-07-02 01:02:05,980 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.41
2021-07-02 01:02:11,029 [app.py:log_request()] - 101 GET /websocket?token=WP6PXFHOKRN2UDGEVJHDYXNKPDMNPPZG (192.168.0.41) [_TRUSTED_USER_] 3.78ms
2021-07-02 01:02:11,030 [websockets.py:add_websocket()] - New Websocket Added: 547568174128
